packages/website/src/components/config.ts

```diff
@@ -1,6 +1,20 @@
-export const detailTabs = [
-  { value: false as const, label: 'Errors' },
-  { value: 'es' as const, label: 'ESTree' },
-  { value: 'ts' as const, label: 'TypeScript' },
-  { value: 'scope' as const, label: 'Scope' },
+import type { ConfigFileType, ConfigShowAst } from './types';
+
+export const detailTabs: { value: ConfigShowAst; label: string }[] = [
+  { value: false, label: 'Errors' },
+  { value: 'es', label: 'ESTree' },
+  { value: 'ts', label: 'TypeScript' },
+  { value: 'scope', label: 'Scope' },
+];
+
+export const fileTypes: ConfigFileType[] = [
+  'ts',
+  'tsx',
+  'js',
+  'jsx',
+  'd.ts',
+  'cjs',
+  'mjs',
+  'cts',
+  'mts',
 ];
```

Comment:
could this be `Object.values(ts.Extension)` instead?

Reply:
a tthis point we do not have access to typescript instance yet as editor may be not loaded